isis/src/base/apps/shadow/ShadowFunctor.cpp

```diff
@@ -393,15 +393,15 @@ namespace Isis {
 
           Hillshade hillshade(azimuth, elevation, elevationModelProjection->Resolution());
 
-          Portal portal(3, 3, m_inputDem->pixelType(), -1.5, -1.5);
-          portal.SetPosition(sample + 1, line + 1, input.Band());
+
+          Portal portal(3, 3, m_inputDem->pixelType(), 0, 0);
```

Comment:
Should these be 0 or the default -0.5, -0.5 so that the point of interest is the start of the pixel?

Reply:
Makes sense, changes to changeling and pixel offset should be updated.